src/main/java/io/vertx/core/http/impl/ws/WebSocketCloseFrameCode.java

```diff
@@ -0,0 +1,37 @@
+package io.vertx.core.http.impl.ws;
+
+import io.netty.buffer.ByteBuf;
+import io.netty.buffer.Unpooled;
+
+import java.nio.charset.Charset;
+
+/**
+ * @author Francesco Guardiani @slinkydeveloper
+ */
+public enum WebSocketCloseFrameCode {
```

Comment:
I don't think we need a special enum for that, can you make a single static method on HttpUtils instead ?

Reply:
The initial idea was to provide a enum with status code. I'm removing it